src/main/webapp/app/exercises/modeling/manage/modeling-exercise-detail.component.ts

```diff
@@ -49,6 +50,18 @@ export class ModelingExerciseDetailComponent implements OnInit, OnDestroy {
         });
     }
 
+    downloadAsPDf() {
+        const model = this.modelingExercise.sampleSolutionModel;
```

Comment:
Is it intended that you use the sample solution here?

Reply:
Yes, that is the idea of export button in detail page.